mbedtls/src/lib.rs

```diff
@@ -7,6 +7,11 @@
  * according to those terms. */
 
 #![deny(warnings)]
+
+#![allow(stable_features)]
+#![feature(universal_impl_trait)]
+#![feature(try_from)]
```

Comment:
This is not acceptable. Why is this needed?

Reply:
Because the old nightly used in core_io build requires a feature for try_from https://travis-ci.com/fortanix/rust-mbedtls/jobs/217291163#L1170 and because
```
#![cfg_attr(feature = "core_io", feature(universal_impl_trait))]
#![cfg_attr(feature = "core_io", feature(try_from))] 
```
doesn't work. Obviously feature doesn't work on stable so yeah wtf how to make this work?

Reply:
Can you update the nightly that's used along with the core_io crate?

Reply:
ok